src/i18n/cy.json

```diff
@@ -34,6 +34,7 @@
     "product_name": "StatsCymru F3.0",
     "beta": "Beta",
     "feedback": "Rydych chi'n edrych ar fersiwn newydd o StatsCymru.  Gallwch gael cymorth trwy droi at <a class=\"govuk-link\" href=\"mailto:{{support_email}}\">{{support_email}}</a>.",
+    "logo": "StatsCymru logo",
```

Comment:
I think @j-maynard prefers to omit new stuff from the Welsh for when he's collecting stuff for the translators. Best to check.